qt/applications/workbench/workbench/plotting/plotscriptgenerator/__init__.py

```diff
@@ -65,7 +65,9 @@ def generate_script(fig, exclude_headers=False):
         axes.legend().set_draggable(True)     (if legend present, or just draggable() for earlier matplotlib versions)
         <Legend title and label commands if non-default>
 
-        plt.show()
+        fig.show()
+        # User plt.show() if running the script outside of Workbench
```

Comment:
I guess there is a typo in the comment where `User` need to be `Use`. Unfortunately that change requires update to unit tests as well.